myInstagram/myInstagram/NotificationViewController/Controller/NotificationTableViewController.swift

```diff
@@ -0,0 +1,170 @@
+//
+//  NotificationTableViewController.swift
+//  myInstagram
+//
+//  Created by Илья on 20.10.2022.
+//
+
+import UIKit
+
+/// Экран уведомления
+final class NotificationTableViewController: UITableViewController {
+    
+    // MARK: - Constants
+    enum Constans {
+        enum ActionsText {
+            static let comment1 = "прокоментировал(-a) ваше фото: Сильный"
+            static let comment2 = "упомянул(-а) вас в комментарии: @alex_ledenev"
+            static let subscribe = "подписался(-ась) на ваши обновления"
+        }
+        
+        enum AvatarImageNames {
+            static let ilentiy = "MyPhoto"
+            static let ledenev = "ledenevAvatar"
+            static let mem = "memAvatar"
+        }
+        enum HeadersText {
+            static let today = "Сегодня "
+            static let week = "На этой неделе "
+            static let mounth = "В этом месяце "
+            
+        }
+        enum Identifiers {
+            static let comment = "Comment"
+            static let subscribe = "Subscribe"
+        }
+        enum NickNames {
+            static let ilentiy = "ilentiy"
+            static let ledenev = "alex_ledenev"
+            static let mem = "memolog"
+        }
+    }
+    
+    private enum TableCellTypes {
+        case today
+        case thisWeek
+    }
+    
+    private enum ActionsItem {
+        case today(items: [Action])
+        case thisWeek(items: [Action])
+    }
+    
+    // MARK: - Private Property
+    private let refresherControl = UIRefreshControl()
+    private let tableCellTypes: [TableCellTypes] = [.today, .thisWeek]
+    
+    private var users: [User] = [
+        User(name: Constans.NickNames.ilentiy, avatar: Constans.AvatarImageNames.ilentiy),
+        User(name: Constans.NickNames.mem, avatar: Constans.AvatarImageNames.mem),
+        User(name: Constans.NickNames.ledenev, avatar: Constans.AvatarImageNames.ledenev)
+    ]
+    
+    private lazy var todayActions: [Action] = [
+        Comment(user: users[1], action: Constans.ActionsText.comment1, post: "ledenevContent1", date: "2 ч."),
```

Comment:
```suggestion
        Comment(user: , action: Constans.ActionsText.comment1, post: "", date: ""),
```
1) обращение по индексу небезопасно
2) Все текстовки в константах

ниже также все поправить